assets/js/Sign.jsx

```diff
@@ -37,6 +37,15 @@ function makeConfig(mode) {
   return { mode: 'off', expires: null };
 }
 
+function displayName(mode) {
```

Comment:
Minor comment (I'm about to approve the PR anyway), but I think it might even be possible to have one map that's used to both display the mode and build the options for the dropdown, which might be kind of slick (if it's not too difficult).